packages/sync-repo-settings/src/required-checks.json

```diff
@@ -4,14 +4,15 @@
     "enableRebaseMerge": false,
     "requireUpToDateBranch": false,
     "requiredStatusChecks": [
-      "Kokoro - Test: Binary Compatibility",
-      "Kokoro - Test: Code Format",
-      "Kokoro - Test: Dependencies",
+      "dependencies",
+      "linkage-monitor",
+      "lint",
+      "clirr",
+      "units (7)",
+      "units (8)",
+      "units (11)",
       "Kokoro - Test: Integration",
-      "Kokoro - Test: Java 11",
-      "Kokoro - Test: Java 7",
-      "Kokoro - Test: Java 8",
-      "Kokoro - Test: Linkage Monitor",
+      "Kokoro - Test: Samples",
```

Comment:
Let's not include this one globally yet.

Reply:
done